# Timeout for joystick reset (<0 no timeout)
joy_msg_timeout: -1
